backend/src/main/scala/cromwell/backend/io/WorkflowPaths.scala

```diff
@@ -1,4 +1,4 @@
-package cromwell.backend.impl.local
+package cromwell.backend.io
```

Comment:
I think we could break out `JobPaths` into its own file. It's confusing to have both WorkflowPaths and JobPaths in `WorkflowPaths.scala`